benchmarks/basic/gocql/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang
```

Comment:
All other benchmarks are `FROM ubuntu:18.04`

I tried to keep it this way to make sure that different base images don't influence performance

But as long as go doesn't beat our driver I guess it doesn't matter

Reply:
Last version of go compiler in ubuntu 18.04 apt repos is from 2018 and does not support go modules. Let's stay with `FROM golang`

Reply:
Sounds reasonable. 
Maybe we should just update other benchmarks to use latest compiler versions as well.